template/utils/evm-account.js

```diff
@@ -0,0 +1,31 @@
+const help = `--evm-account <AccountId32> | <H160>: Calculate the EVM gas charge account that corresponds to a native Substrate Account or EVM address.`;
+
+module.exports = () => {
+  if (process.argv.length < 4) {
+    console.error('Please provide the <account> or <address> parameter.');
+    console.error(help);
+    process.exit(9);
+  }
+
+  const address = process.argv[3];
+  if (!address.match(/^[A-z0-9]{40}|0x[A-z0-9]{40}|[A-z0-9]{48}$/)) {
```

Comment:
I think using `[A-Fa-f0-9]{40}` for evm addresses is better here?

Reply:
good suggestion